atom/epistula/README.md

```diff
@@ -0,0 +1,75 @@
+# Implementation of the epistula protocol
+Official documentation for the epistula protocol can be found [here](https://epistula.sybil.com/).
+
+## Quick Start
```

Comment:
I would like a description about why you would even do this. Code seems simple enough but I think going to their docs lack some verbosity that could be useful.